docs/Makefile

```diff
@@ -1,9 +1,9 @@
-.PHONY: default docs docs-build docs-shell test
+.PHONY: default docs docs-draft docs-shell test docs-build
 
 # to allow `make DOCSDIR=docs docs-shell` (to create a bind mount in docs)
 DOCS_MOUNT := $(if $(DOCSDIR),-v $(CURDIR)/$(DOCSDIR):/$(DOCSDIR))
 
-# to allow `make DOCSPORT=9000 docs`
+# to allow `make DOCSPORT=8000 docs`
```

Comment:
this is to illustrate you can use a different port than the default (`8000`), so `make DOCSPORT=9000 docs` would result in the server running on port 9000 instead of 8000